src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+                        logger.info(
+                            f"""
+{termColors.FAIL}
+!!! ERROR: duplicate entries exist in the
+transtion_domain table for domain:
+{new_entry_domainName}
+----------TERMINATING----------"""
+                        )
+                        import sys
+
+                        sys.exit()
+
+                # DEBUG:
+                if debug_on or debug_max_entries_to_parse > 0:
+                    if (
+                        total_rows_parsed > debug_max_entries_to_parse
+                        and debug_max_entries_to_parse != 0
+                    ):
+                        logger.info(
+                            f"""{termColors.WARNING}
+                            ----PARSE LIMIT REACHED.  HALTING PARSER.----
+                            {termColors.ENDC}
+                            """
+                        )
+                        break
+
+        TransitionDomain.objects.bulk_create(to_create)
+
+        logger.info(
+            f"""{termColors.OKGREEN}
+
+        ============= FINISHED ===============
+        Created {total_new_entries} transition domain entries,
+        updated {total_updated_domain_entries} transition domain entries
+        {termColors.ENDC}
+        """
+        )
+
+        # Print a summary of findings (duplicate entries,
+        # missing data..etc.)
+        self.print_summary_duplications(
+            duplicate_domain_user_combos, duplicate_domains, users_without_email
+        )
+        self.print_summary_status_findings(domains_without_status, outlier_statuses)
+
+    def print_debug_mode_statements(self, debug_on, debug_max_entries_to_parse):
+        if debug_on:
+            logger.info(
+                f"""{termColors.OKCYAN}
+            ----------DEBUG MODE ON----------
+            Detailed print statements activated.
+            {termColors.ENDC}
+            """
+            )
+        if debug_max_entries_to_parse > 0:
+            logger.info(
+                f"""{termColors.OKCYAN}
+            ----------LIMITER ON----------
+            Parsing of entries will be limited to
+            {debug_max_entries_to_parse} lines per file.")
+            Detailed print statements activated.
+            {termColors.ENDC}
+            """
+            )
+
+    def get_domain_user_dict(self, domain_statuses_filename, sep):
+        """Creates a mapping of domain name -> status"""
+        # TODO: figure out latest status
+        domain_status_dictionary = defaultdict(str)
+        # NOTE: how to determine "most recent" status?
+        logger.info("Reading domain statuses data file %s", domain_statuses_filename)
+        with open(domain_statuses_filename, "r") as domain_statuses_file:  # noqa
+            for row in csv.reader(domain_statuses_file, delimiter=sep):
+                domainName = row[0].lower()
+                domainStatus = row[1].lower()
+                # print("adding "+domainName+", "+domainStatus)
```

Comment:
**suggestion (non-blovking)**: Again, we don't want to keep debug print statements commented out in the code. Either use `logger.info` if you want to keep it, or delete the commented out line.
```suggestion
```